tests/test_sbi.py

```diff
@@ -698,6 +938,294 @@ def simulator(params):
     # -------
     # Run validation
 
-    ValidationRunner.from_config("./toy/val.yml")
+    val_runner = ValidationRunner.from_config("./toy/val.yml")
+    val_runner(loader=loader)
+    
+    val_runner = ValidationRunner.from_config("./toy/val_vi.yml")
+    # val_runner(loader=loader)
```

Comment:
Should these be uncommented?

Reply:
These were commented as those metrics were taking a very long time when running the tests. I have now realised this was due to using too many validations simulations, so I have reduced this number and uncommented these lines.